web/shared/src/typing_status.ts

```diff
@@ -110,6 +166,17 @@ export let actually_ping_server = (
     set_next_start_time(current_time, typing_started_wait_period);
 };
 
+function actually_ping_server_for_message_edit(
+    worker: EditingStatusWorker,
+    message_id: number,
+    current_time: number,
+    typing_started_wait_period: number,
+): void {
+    const state = editing_state.get(message_id)!;
```

Comment:
We should assert here, if that's expected to always be true. Though it might be helpful to have a comment explaining why.